tsl/test/shared/sql/CMakeLists.txt

```diff
@@ -24,6 +24,10 @@ foreach(TEMPLATE_FILE ${TEST_TEMPLATES_SHARED})
   list(APPEND TEST_FILES_SHARED ${TEST_FILE})
 endforeach(TEMPLATE_FILE)
 
+if(DEFINED EXT_GIT_COMMIT_TAG OR DEFINED EXT_GIT_COMMIT_HASH OR DEFINED EXT_GIT_COMMIT_TIME)
```

Comment:
Likely checking the GIT_FOUND should be enough here

Reply:
Unfortunately not. If git is found but we're not in a git repository, `GIT_FOUND` will be true, but this test will generate an error because `get_git_commit` will generate an error if neither of these fields are found.